plugins/fix-visibility/plugin.go

```diff
@@ -37,6 +38,8 @@ func main() {
 
 // FixVisibilityPlugin implements an aspect CLI plugin.
 type FixVisibilityPlugin struct {
+	plugin.Base
```

Comment:
This plug-in has nothing that can be deleted now?

(sorry on mobile add hard to tell atm...)

Reply:
Nope